### ISSUE TITLE
No GUI or front end

### ISSUE THREAD

**Issue by Solasoul** [elementary/mail#798]
### What Happened?

When you add an account, there is no GUI or front end to access your mail. The Account is just listed.

### Steps to Reproduce

Click the Mail app and add your email account, and there's no other option or any notifications. The app just collects the information.

### Expected Behavior

The App should open up to the inboxes and folders and function like an email app

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
There is no crash information, the app just doesn't function correctly
```

### Hardware Info

Apple iMac computer 8 GB RAM

**Comment by danrabbit** [comment]
Can you post a screenshot of what you're seeing?

**Comment by Solasoul** [comment]
Hi, Sorry I no longer have it installed but it just didn't load maybe it was the internet connection. Can you please try and run I was on the latest Elementary.

**Comment by janxkoci** [comment]
@danirabbit I think they were seeing the Online Accounts plug in Switchboard, but were not redirected back to the Mail app after setting up the account.

**Comment by danirabbit** [comment]
It's possible this was fixed by #855 but without a screenshot or some further description of what was being seen, we can't really know. I'm going to assume this was fixed and close it as completed, but if this means something else, feel free to reopen
